features/game/components/QrCode/PdfTemplate.tsx

```diff
@@ -0,0 +1,36 @@
+import { CSSProperties } from 'react'
+
+export const containerStyle: CSSProperties = {
+  minWidth: '210mm',
+  minHeight: '297mm',
+  backgroundColor: 'white',
+  position: 'fixed',
+  top: '0',
+  left: '100vw',
+}
```

Comment:
Please use TailwindCSS for styling

Reply:
https://github.com/re-quest-team/request/pull/57#discussion_r894364853